app/views/shared/_collapsible.html.erb

```diff
@@ -1,11 +1,21 @@
 <div class="collapsible">
   <h3 class="collapsible__heading">Do you know what the recommended amount per week a student should spend on food?</h3>
-  <a class="collapsible__link" href="">
-    Reveal the answer
-    <svg class="collapsible__icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0" y="0" viewBox="10 10 80 80" enable-background="new 0 0 100 100" xml:space="preserve">
-      <circle fill="#0076AB" cx="50" cy="50" r="36"/>
-      <line fill="none" stroke="#FFFFFF" stroke-width="6" stroke-miterlimit="10" x1="50" y1="35" x2="50" y2="65"/>
-      <line fill="none" stroke="#FFFFFF" stroke-width="6" stroke-miterlimit="10" x1="35" y1="50" x2="65" y2="50"/>
+  <div class="collapsible__reveal" data-dough-collapsable-target="answer_reveal">
+    <p>Bacon ipsum dolor amet short ribs bresaola pig venison kielbasa corned beef swine boudin ball tip ribeye tail porchetta sirloin alcatra. Fatback frankfurter brisket, kevin doner biltong cupim cow sausage ham chuck. Shankle turkey ball tip, bacon picanha short loin pork loin spare ribs frankfurter beef ribs. Filet mignon andouille strip steak pork loin cupim capicola bresaola biltong chuck bacon. Shank ground round fatback porchetta shankle sirloin andouille kevin filet mignon pork belly short loin short ribs spare ribs cow drumstick. Doner cow pig andouille pork, salami tri-tip pork belly leberkas brisket.</p>
```

Comment:
That shared component is only there for the styleguide - the actual implementation of the module is elsewhere in the ruby as a text filter module